migrations/v2.js

```diff
@@ -0,0 +1,115 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+const getCourse = content => {
+  const course = content.find(({ _type }) => _type === 'course');
+  return course;
+};
+
+const getGlobals = content => {
+  return getCourse(content)?._globals?._extensions?._pageLevelProgress;
+};
+
+describe('Page level progress - v2.0.1 to v2.0.2', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/compare/v2.0.1..v2.0.2
+
+  let course, coursePlpGlobals;
+  const plpIndicatorBarDefaultOld = 'You have completed ';
+  const plpIndicatorBarDefaultNew = 'Progress bar. Select here to view your current progress, and select an item to navigate to it. You have completed ';
+
+  whereFromPlugin('Page level progress - from v2.0.1', { name: 'adapt-contrib-pageLevelProgress', version: '<2.0.2' });
+
+  whereContent('Page level progress is configured', content => {
+    course = getCourse(content);
```

Comment:
Standard change for all getCourse function calls with the new helpers
```suggestion
    course = getCourse();
```